thinkhazard/__init__.py

```diff
@@ -18,6 +19,7 @@ def main(global_config, **settings):
     config = Configurator(settings=settings)
 
     config.include('pyramid_jinja2')
+    config.include('papyrus.includeme')
```

Comment:
Use `config.include('papyrus')` for consistency with the way we include `pyramid_jinja2`.